chore: update project config

Aleks334 opened the pull request:
Adds prettier config globally and sets absolute path on frontend.

Aleks334 committed:
chore(vite): setup absolute path

Aleks334 committed:
chore: setup prettier

Aleks334 committed:
style: format project files

mit3r reviewed (commented):
Great job, if you allow me I will add similar changes on backend. Let me know if it's ok in your opinion

Commit:
adds json formating

Commit:
add absolute paths on backend

Aleks334 committed:
chore: add @global alias to client's tsconfig

Aleks334 committed:
fix: add watch in server's package.json script along with import change

Aleks334 commented:
Yeah. It's ok. I added global alias also in client. Moreover I introduced small fix in one of server's imports and using watch with tsx in it's package.json script

mit3r reviewed (approved):
Everything is great, thanks for taking care of it.
